Domains/3-DisciplineOther/Hydraulics/SewerHydraulicAnalysis.ecschema.xml

```diff
@@ -1057,6 +1057,7 @@
 
     <ECEntityClass typeName="TimeOfConcentrationFlowPath" modifier="Sealed" displayLabel="Tc Flow Path">
         <BaseClass>anlyt:AnalyticalElement</BaseClass>
+        <ECProperty propertyName="Geometry" typeName="Bentley.Geometry.Common.IGeometry" displayLabel="Flow Path Geometry"/>
```

Comment:
The name of this property should be less generic so it is easier to understand by any readers reviewing this class. How about "FlowPath"?